catalogs/catalog-lakehouse-paimon/src/test/java/org/apache/gravitino/catalog/lakehouse/paimon/TestGravitinoPaimonTable.java

```diff
@@ -389,7 +458,7 @@ void testGravitinoToPaimonTable() {
     Assertions.assertEquals(new IntType().nullable(), paimonTableSchema.fields().get(0).type());
     Assertions.assertEquals(new DateType().nullable(), paimonTableSchema.fields().get(1).type());
     Assertions.assertEquals(
-        new VarCharType(Integer.MAX_VALUE).nullable(), paimonTableSchema.fields().get(2).type());
+        new VarCharType(Integer.MAX_VALUE).notNull(), paimonTableSchema.fields().get(2).type());
```

Comment:
why changing this line?

Reply:
https://github.com/apache/gravitino/pull/4292/files#diff-ae0fe81137e01313281a32065109a682c249928f6e91aa331f73abcc070f5668R611-R613
I changed col3 to notnull to make it easier to create a primary key.
it is affected by this change

Reply:
got it